wesnoth-utbs.po

```diff
@@ -8842,60 +8802,43 @@ msgid ""
 "desert sun."
 msgstr ""
 "Kalehu, já jsem Eloh, nositelka Ištařiny hole a ta, která zabila boha-démona "
-"Zhangora. Udělej to, co ti říkám! Vzdej se mu nebo opustím tvůj lid a "
-"zanechám jej napospas utrpení a smrti. Vaše kosti posypou písečné duny a "
-"supi se budou krmit vaším masem. Nejsem promíjející bohyně, Kalehu."
+"Zhangora. Vzdej se mu nebo opustím tvůj lid a zanechám jej napospas utrpení "
+"a smrti. Vaše kosti posypou písečné duny a supi se budou krmit vaším masem. "
+"Jsem dobrá bohyně, Kalehu, ale neodpouštím víc než ostré pouštní slunce."
 
 #. [message]: speaker=Kaleh
 #: data/campaigns/Under_the_Burning_Suns/scenarios/08_Out_of_the_Frying_Pan.cfg:3482
-#, fuzzy
-#| msgid ""
-#| "Then kill me if you must, but I will not give myself over to those who "
-#| "threaten me and my people. I have not come through peril and darkness "
-#| "just to surrender to a man such as you."
 msgid ""
 "Then kill me if you must, but I will not give myself over to those who "
 "threaten me and my people. I have not come through peril and darkness just "
 "to surrender to one such as you."
 msgstr ""
-"Pak mě zabij, jestli musíš, ale nevzdám se tomu, kdo vyhrožuje mi a mému "
-"lidu. Neprošel jsem utrpením a temnotou jenom proto, abych se vzdal takovému "
-"člověku jako jsi ty."
+"Pak mě zabij, jestli musíš. Ale nevzdám se tomu, kdo vyhrožuje mě a mému "
+"lidu. Neprošel jsem utrpením a temnotou jenom proto, abych se vzdal někomu "
+"takovému jako jsi ty."
 
 #. [message]: speaker=Sergeant Durstrag
 #: data/campaigns/Under_the_Burning_Suns/scenarios/08_Out_of_the_Frying_Pan.cfg:3487
-#, fuzzy
-#| msgid ""
-#| "Your dare defy me?! All who refuse to submit to the authority of the Iron "
-#| "Council shall be killed. By the Dark Lady, I shall not put up with this "
-#| "bickering any longer. To battle, men! Drive those heathens back into the "
-#| "caves!"
 msgid ""
 "You dare defy me?! All who refuse to submit to the authority of the Iron "
 "Council shall be killed. By the Dark Lady, I shall not put up with this "
 "bickering any longer. To battle, men! Drive those heathens back into the "
 "caves!"
 msgstr ""
-"Ty si troufáš pohrdat mnou?!? Každý, kdo se odmítne podřídit autoritě "
-"Železné rady, má být zabit. U Temné paní, nebudu se obtěžovat s tímhle "
-"dohadováním ani o chvíli déle. Muži, do boje! Zatlačte ty neznabohy zpátky "
-"do jeskyní!"
+"Ty se mi opovažuješ vzdorovat?! Každý, kdo se odmítne podřídit autoritě "
+"Železné rady, má být zabit. U Temné paní, nebudu už zdržovat tímhle "
```

Comment:
Bez toho **se** mi to zní skoro tak, jako by se omlouval. Ale možná to špatně chápu.
```suggestion
"Železné rady, má být zabit. U Temné paní, nebudu se už zdržovat tímhle "
```